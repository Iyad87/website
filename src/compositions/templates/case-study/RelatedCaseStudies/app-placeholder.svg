<?xml version="1.0" encoding="UTF-8"?>
<svg width="320px" height="411px" viewBox="0 0 320 411" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 50.2 (55047) - http://www.bohemiancoding.com/sketch -->
    <title>Bitmap Copy 2</title>
    <desc>Created with Sketch.</desc>
    <defs>
        <rect id="path-1" x="0" y="0" width="486" height="333" rx="4"></rect>
        <filter x="-9.4%" y="-11.6%" width="118.7%" height="127.3%" filterUnits="objectBoundingBox" id="filter-3">
            <feMorphology radius="2.5" operator="dilate" in="SourceAlpha" result="shadowSpreadOuter1"></feMorphology>
            <feOffset dx="0" dy="7" in="shadowSpreadOuter1" result="shadowOffsetOuter1"></feOffset>
            <feGaussianBlur stdDeviation="11.5" in="shadowOffsetOuter1" result="shadowBlurOuter1"></feGaussianBlur>
            <feColorMatrix values="0 0 0 0 0.631372549   0 0 0 0 0.62745098   0 0 0 0 0.77254902  0 0 0 0.18 0" type="matrix" in="shadowBlurOuter1"></feColorMatrix>
        </filter>
    </defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="CASE-STUDY-STATIC" transform="translate(-960.000000, -7347.000000)">
            <g id="Bitmap-Copy-2" transform="translate(640.000000, 7348.000000)">
                <g id="Group-18" transform="translate(350.000000, 40.000000)">
                    <mask id="mask-2" fill="white">
                        <use xlink:href="#path-1"></use>
                    </mask>
                    <g id="Mask">
                        <use fill="black" fill-opacity="1" filter="url(#filter-3)" xlink:href="#path-1"></use>
                        <use fill="#FFFFFF" fill-rule="evenodd" xlink:href="#path-1"></use>
                    </g>
                    <rect id="Rectangle-8" fill="#E4E8EF" mask="url(#mask-2)" x="-6" y="-12" width="536" height="36"></rect>
                    <circle id="Oval-3" fill="#F3F5FA" mask="url(#mask-2)" cx="14" cy="12" r="4"></circle>
                    <circle id="Oval-3" fill="#F3F5FA" mask="url(#mask-2)" cx="26" cy="12" r="4"></circle>
                    <circle id="Oval-3" fill="#F3F5FA" mask="url(#mask-2)" cx="38" cy="12" r="4"></circle>
                </g>
            </g>
        </g>
    </g>
</svg>