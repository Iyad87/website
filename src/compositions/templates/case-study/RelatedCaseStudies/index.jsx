import React from 'react';
import TileGrid, { Tile } from 'components/TileGrid';

import {
  Section,
  TileContent,
  TileContentTitle,
  TileContentBody,
  Image,
  Right,
  Left,
} from './styled';

import phoneImage from './phone-placeholder.svg';
import placeholder from './app-placeholder.svg';
import clientLogo from './trainline-logo.png';
import secondClientLogo from './joyent-logo.png';

const RelatedCaseStudies = () => (
  <Section wide>
    <TileGrid>
      <Tile>
        <TileContent>
          <Left>
            <header>
              <TileContentTitle>Related Case Studies</TileContentTitle>
              <img alt="client logo" src={clientLogo} />
            </header>
            <TileContentBody>
              A legacy platform that faced a huge pressure to have an updated
              interface.
            </TileContentBody>
          </Left>
          <Right>
            <Image src={phoneImage} />
          </Right>
        </TileContent>
      </Tile>
      <Tile colour="lightGreen">
        <TileContent>
          <Left>
            <header>
              <TileContentTitle>Related Case Studies</TileContentTitle>
              <img alt="client logo" src={secondClientLogo} />
            </header>
            <TileContentBody>
              A legacy platform that faced a huge pressure to have an updated
              interface.
            </TileContentBody>
          </Left>
          <Right>
            <Image src={placeholder} />
          </Right>
        </TileContent>
      </Tile>
    </TileGrid>
  </Section>
);

export default RelatedCaseStudies;
