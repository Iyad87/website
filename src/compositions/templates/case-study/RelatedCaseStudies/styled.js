import styled from 'styled-components';
import { spacing } from 'styles/spacing';
import Container from 'components/Container';

export const Section = Container.withComponent('section').extend`
  margin-bottom: ${spacing(5)};
  margin-top: ${spacing(15)};
`;

export const TileContent = styled.div`
  display: flex;
`;

export const TileContentTitle = styled.h5`
  letter-spacing: 0.4px;
  font-size: 1.125rem;
  font-weight: 700;
  margin: 0;
  margin-bottom: 10px;
  text-transform: uppercase;
`;

export const TileContentBody = styled.p`
  font-weight: 300;
  font-size: 1.5rem;
  margin: 0;
`;

export const Left = styled.div`
  width: 50%;
  display: flex;
  justify-content: space-between;
  flex-direction: column;
  height: 100%;
  padding: ${spacing(4)};
`;

export const Right = styled.div`
  position: relative;
  width: 50%;
`;

export const Image = styled.img`
  display: block;
  margin-left: auto;
  width: auto;
  height: 100%;
`;
