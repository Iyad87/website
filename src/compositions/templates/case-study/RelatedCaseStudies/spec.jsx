import React from 'react';
import { shallow } from 'enzyme';

import RelatedCaseStudies from './index';

test('RelatedCaseStudies', () => {
  const wrapper = shallow(<RelatedCaseStudies />);

  expect(wrapper).toMatchSnapshot();
});
