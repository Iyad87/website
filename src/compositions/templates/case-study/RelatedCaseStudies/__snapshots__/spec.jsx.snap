// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RelatedCaseStudies 1`] = `
<Container
  wide={true}
>
  <TileGrid>
    <styled__Tile>
      <styled__TileContent>
        <styled__Left>
          <header>
            <styled__TileContentTitle>
              Related Case Studies
            </styled__TileContentTitle>
            <img
              alt="client logo"
              src="/src/compositions/templates/case-study/RelatedCaseStudies/trainline-logo.png"
            />
          </header>
          <styled__TileContentBody>
            A legacy platform that faced a huge pressure to have an updated interface.
          </styled__TileContentBody>
        </styled__Left>
        <styled__Right>
          <styled__Image
            src="/src/compositions/templates/case-study/RelatedCaseStudies/phone-placeholder.svg"
          />
        </styled__Right>
      </styled__TileContent>
    </styled__Tile>
    <styled__Tile
      colour="lightGreen"
    >
      <styled__TileContent>
        <styled__Left>
          <header>
            <styled__TileContentTitle>
              Related Case Studies
            </styled__TileContentTitle>
            <img
              alt="client logo"
              src="/src/compositions/templates/case-study/RelatedCaseStudies/joyent-logo.png"
            />
          </header>
          <styled__TileContentBody>
            A legacy platform that faced a huge pressure to have an updated interface.
          </styled__TileContentBody>
        </styled__Left>
        <styled__Right>
          <styled__Image
            src="/src/compositions/templates/case-study/RelatedCaseStudies/app-placeholder.svg"
          />
        </styled__Right>
      </styled__TileContent>
    </styled__Tile>
  </TileGrid>
</Container>
`;
